### PR TITLE
Wrote Tests for ERC1155

### LINKED ISSUE #599: Create ERC1155 Unity unit tests
From "ERC1155" Demo Scene section create following unity tests:
1. BalanceOf
2. OwnerOfBatch
3. URI
4. Import NFT Texture (if posible)
5. Call the indexer (if possible)

Call the same logic that is called by the buttons in demo scene.
For the Ethereum simulation use `anvil` in a similar manner how it is done here:
https://github.com/ChainSafe/web3.unity/blob/4bae12c11ce06322bef02b4a63ac0bf7b540c850/src/ChainSafe.GamingSDK.EVM.Tests/ChainsafeRPCTests.cs#L30

### PR DESCRIPTION
**Pull request by kantagara**
Fixing linter for windows.
Fixed exception in Unity when first opening the scene (Web3Auth related)

closes #599

### CODE REVIEW

**Review by RyRy79261 — APPROVED** [review]

**Review by sneakzttv — DISMISSED** [review]

File: lint.bat [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 dotnet format --verbosity=d --verify-no-changes --severity=warn .\ChainSafe.Gaming.sln --exclude .\submodules
```
Comment (kalambet) [inline lint.bat:3]:
```suggestion
dotnet format --verbosity=d --severity=warn .\ChainSafe.Gaming.sln --exclude .\submodules
```

File: lint.bat [hunk 2]
```diff
@@ -2,6 +2,6 @@
...
 
-pushd src/UnitySampleProject
+pushd .\src\UnitySampleProject
 dotnet format --verbosity=d --verify-no-changes --severity=warn .\UnitySampleProject.sln
```
Comment (kalambet) [inline lint.bat:6]:
```suggestion
dotnet format --verbosity=d --severity=warn .\UnitySampleProject.sln
```

Comment (kalambet) [reply lint.bat:6]: in order for a script to actually lint the code this argument needs to be removed from both commands

**Review by sneakzttv — APPROVED** [review]

**Review by sneakzttv — APPROVED** [review]

**Review by juans-chainsafe — APPROVED** [review]
the tests looks good! and playmod and webgl builds are working correctly

**Pull request merged**